code/controllers/subsystem/redis.dm

```diff
@@ -0,0 +1,115 @@
+SUBSYSTEM_DEF(redis)
+	name = "Redis"
+	init_order = INIT_ORDER_REDIS
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY // ALL THE THINGS
+	wait = 1
+	flags = SS_TICKER // Every tick
+	/// Are we connected
+	var/connected = FALSE
+	/// Amount of subscribed channels on the redis server
+	var/list/subbed_channels = list()
+	/// Message queue (If messages are sent before the SS has init'd)
+	var/list/datum/redis_message/queue = list()
+	offline_implications = "The server will no longer be able to send or receive redis messages. Shuttle call recommended (Potential server crash inbound)."
```

Comment:
As for why `Potential server crash inbound`, the redis rust module caps at 1000 messages to avoid ram exhaustion (since thats an issue at TG). 

What happens when the limit of 1000 messages is reached? 
Good question.